### PR TITLE
Migrate to EventStream in precaching

### PR DESCRIPTION
**Pull request by Ladsgroup**
Even though it's straightforward, I hadn't have time to test it.
Bug: https://phabricator.wikimedia.org/T166046

### CODE REVIEW

File: ores/utilities/precached.py [hunk 1]
```diff
@@ -66,142 +76,32 @@ def main(argv=None):
...
+            time.sleep(delay)
+        if notify:
+            watchdog_ping(*notify)
+        start = time.time()
+        response = requests.post(
+            ores_url + "/v3/precache/", data=json.dumps(change))
+
+        if response.status == 200:
+            logger.info("Scored {0} in {1} seconds."
+                        .format(json.dumps(change), round(time.time() - start, 3)))
```
Comment (Ladsgroup) [inline ores/utilities/precached.py]: I have a feeling that change will be a huge variable, we should only get rev id.

Comment (halfak) [inline ores/utilities/precached.py]: This is a fair point.  There may be some more important things.  E.g. 'bot', 'namespace', etc.   We should get that small subset or the whole thing.

**Comment by halfak** [comment]
This needs a real world test before merging.

**Comment by Ladsgroup** [comment]
Okay, I made lots of tests and made tons of fixes. It now works but server responds with "database" is not there (and "rev_id" is not there) because change schema of event stream is different than what is change propagation change scheme (ping @ottomata and @Pchelolo, Any plans to make them look alike?)

Also this needs to send user-agent.

**Comment by ottomata** [comment]
Ya you are reading recent changes which is s different stream. We've been waiting for someone to need revision-create in the EventStreams public API before we added it in. Looks like this is it! :)

**Comment by halfak** [comment]
Yeah!  please :)))

**Comment by halfak** [comment]
See https://phabricator.wikimedia.org/T167670

**Comment by halfak** [comment]
tested on my local machine.  I ran this command to set up precached daemon:

```
./utility precached https://stream.wikimedia.org/v2/stream/revision-create http://localhost:8080
```

**Pull request merged**